src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -1,10 +1,36 @@
+import { useState } from 'react';
+import CountryInfo from '../CountryInfo/CountryInfo';
 import { ChatHeaderStyled } from './ChatHeaderStyled';
 
-const ChatHeader = ({ countryName, participantsAmount }) => (
-  <ChatHeaderStyled>
-    <h5>{countryName || 'Country Name'}</h5>
-    <p>{`${participantsAmount || '0'} participants`}</p>
-  </ChatHeaderStyled>
-);
+const ChatHeader = ({
+  countryName,
+  participantsAmount,
+  countryChatId,
+  setSubscriptionCountryRooms,
+  isSubscribed,
+}) => {
+  const [openModal, setOpenModal] = useState(false);
+  const handleOpen = () => setOpenModal(true);
+  const handleClose = () => {
+    setOpenModal(false);
+  };
+  return (
+    <>
+      <ChatHeaderStyled onClick={handleOpen}>
```

Comment:
It would be nice to have some hover state so that user can understand that he is able to click here.
![image](https://github.com/user-attachments/assets/c8cee4db-9edc-43ee-9202-b06d03bc2018)

Reply:
Done. Add change color text and cursor pointer